incubator/openfaas/README.md

```diff
@@ -0,0 +1,68 @@
+## Deploy OpenFaaS via Helm
+
+**Note:** You must also pass `--set rbac=false` if your cluster is not configured with role-based access control. For further information, see [here](https://kubernetes.io/docs/admin/authorization/rbac/).
+
+---
+
+Clone the faas-netes repo including the helm chart:
```

Comment:
Isn't this section no longer needed if it's in the charts repo?

Reply:
I didn't raise this PR.. @zouyee can you comment?

Reply:
@alexellis  got it, I will fix it later.